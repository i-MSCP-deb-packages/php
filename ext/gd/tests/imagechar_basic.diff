001+ 8fdd650ab9fa03b3628fca1b48230334
001- e94962ac28ad03bd4142cb1abe9ef98b