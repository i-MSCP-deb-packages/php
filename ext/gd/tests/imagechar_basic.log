
---- EXPECTED OUTPUT
e94962ac28ad03bd4142cb1abe9ef98b
---- ACTUAL OUTPUT
8fdd650ab9fa03b3628fca1b48230334
---- FAILED
