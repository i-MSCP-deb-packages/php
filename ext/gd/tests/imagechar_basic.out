8fdd650ab9fa03b3628fca1b48230334